islandora.module

```diff
@@ -58,7 +58,7 @@ define('ISLANDORA_DATASTREAM_INGESTED_HOOK', 'islandora_datastream_ingested');
 define('ISLANDORA_DATASTREAM_MODIFIED_HOOK', 'islandora_datastream_modified');
 define('ISLANDORA_DATASTREAM_PURGED_HOOK', 'islandora_datastream_purged');
 define('ISLANDORA_INGEST_STEP_HOOK', 'islandora_ingest_steps');
-define('ISLANDORA_DERVIATIVE_CREATION_HOOK', 'islandora_derivative');
```

Comment:
To properly deprecate this, we need to continue creating the old constant (possibly pointing to the "new" constant), with a comment indicating that it should be removed after the next release... 7.x-1.6 or so?).